### ISSUE TITLE
Pause host to let party catch up?

### ISSUE THREAD

**Issue by sm11963** [samcm/synclounge#46]
So before I used Plextogether, I used Showgoers for Netflix. One thing I think Showgoers does well is keep the players very in sync. One technique they use is to pause playback if any client is lagging behind.

I think this strategy would be something PlexTogether would benefit from. At the very least as an optional synchronization strategy.

### Example
Say we have a Host and a Member in the party.

- Host has good internet connection, never needs to stop to buffer
- Member has poor internet connection, frequently needs to stop to buffer

What seems to happen right now is that playback is constantly choppy for the Member while the Host sees no issues and will not realize the Member is experiencing problems. 

In this case it might be a better experience for everyone to pause playback to allow all members to catch up (thus not requiring skipping any playback for the members). While the Host now has a worse experience, no party member will have to miss scenes and can watch more in sync.

**Comment by samcm** [comment]
I think this is similar to #37 yeah? 

At the moment the app is structured around one source of truth (the host) and changing away from this would require significant changes to other features like autoplay and seeking. 

If it was implemented, the actual sync result wouldn't change - we're still hamstrung by the Plex Client and how laggy/feature-rich they are. That is to say, the benefit from this would solely be that anyone can play/pause etc. Other apps like Showgoers and Syncplay all have a consistent player that they can work around. Using the PT Player we created allows for a much much better sync but it obviously won't help if someone is lagging.

Personally I don't think it's worth the time investment but I'll keep an open mind if someone manages to convince me or make a PR :P
